### DISCUSSION TITLE
Local port access is denied

### DISCUSSION THREAD

**Discussion by shatangxiaoyan** [Difegue/LANraragi#1072] [Q&A] [answered]
The system version is Windows 10, and WSL is running on WSL1. The issue shows that local plus port access is denied. Upon opening the log, it says that the system cannot find the specified path
<img width="734" alt="屏幕截图 2024-09-03 095920" src="https://github.com/user-attachments/assets/14ae4ee9-23b5-4cfd-bafc-ff0579f7a55f">

**Comment by Difegue** [comment, marked as answer]
WSL1 is deprecated, I'd recommend moving to 2 if possible.
